### PR TITLE
✨ Read windows appx packages on a FS scan.

### PR DESCRIPTION
**Pull request by preslavgerchev**
Support looking for appx packages on a windows FS scan. We look for `appxmanifest.xml` files and parse those to determine the package info.

### CODE REVIEW

File: providers/os/resources/packages/windows_packages.go [hunk 1]
```diff
@@ -38,20 +41,20 @@ const (
 // https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ff357803(v=vs.85)
 var (
 	wsusClassificationGUID = map[string]WSUSClassification{
-		"5c9376ab-8ce6-464a-b136-22113dd69801 ": Application,
```
Comment (chris-rock) [inline providers/os/resources/packages/windows_packages.go:41]: Thank you for the cleanup

File: providers/os/resources/packages/appx_manifest.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package packages
+
+import "encoding/xml"
+
+// AppxManifest represents the structure of an AppxManifest.xml file
+type AppxManifest struct {
```
Comment (chris-rock) [inline providers/os/resources/packages/appx_manifest.go]: I would like to see a test here where we parse a manifest file in go test.

File: providers/os/resources/packages/windows_packages.go [hunk 3]
```diff
@@ -268,6 +283,30 @@ func (w *WinPkgManager) getInstalledApps() ([]Package, error) {
...
 }
 
+func (w *WinPkgManager) getAppxPackages() ([]Package, error) {
+	canRunCmd := w.conn.Capabilities().Has(shared.Capability_RunCommand)
+	// we always prefer to use the powershell command to get the appx packages, fallback to filesystem if not possible
+	if !canRunCmd && (w.conn.Type() == shared.Type_FileSystem || w.conn.Type() == shared.Type_Device) {
+		return w.getFsAppxPackages()
+	}
+
+	b, err := windows.Version(w.platform.Version)
```
Comment (chris-rock) [inline providers/os/resources/packages/windows_packages.go:293]: I think we want to create a seperate function like `getPwshAppxPackages()`. Then the function is easier to read.

**Review by chris-rock — COMMENTED** [review]
Great addition @preslavgerchev Lets add some tests and we are good to go.

File: providers/os/resources/packages/appx_manifest.go (hunk 2 above)
Comment (preslavgerchev) [reply providers/os/resources/packages/appx_manifest.go]: done

File: providers/os/resources/packages/windows_packages.go (hunk 3 above)
Comment (preslavgerchev) [reply providers/os/resources/packages/windows_packages.go:293]: good point, done

File: providers/os/resources/packages/appx_manifest.go (hunk 2 above)
Comment (preslavgerchev) [reply providers/os/resources/packages/appx_manifest.go]: also added a test for `toPackage` which convers the appxPackage to just a package

**Pull request merged**